proc_scripts/load_data.py

```diff
@@ -95,43 +243,92 @@ def proc_Hahn_echoph(s, fl=None):
         fl.image(abs(s))
     return s
 
-def proc_spincore_IR(s,fl=None):
+def proc_spincore_IR(s,clock_correction,fl=None):
+    logger.info(strm("pre-processing for IR acquired on spincore"))
     s.rename('vd','indirect')
     s.reorder(['ph1','ph2','indirect','t2'])
     s.ft(['ph2','ph1'])
     s.ft('t2', shift=True)
     if fl is not None:
         fl.next('raw data -- coherence channels')
-        fl.image(s)
+        fl.image(s.C.setaxis('indirect','#').set_units('indirect','scan #'))
     s.ift('t2')
     if fl is not None:
         fl.next('time domain (all $\\Delta p$)')
-        fl.image(s)
+        fl.image(s.C.setaxis('indirect','#').set_units('indirect','scan #'))
     s.ft('t2', pad=4096)
     if fl is not None:
         fl.next('frequency domain (all $\\Delta p$)')
-        fl.image(s)
+        fl.image(s.C.setaxis('indirect','#').set_units('indirect','scan #'))
+    s *= exp(-1j*s.fromaxis('indirect')*clock_correction)
     return s
 
-def proc_nutation(s):
-    logging.info("loading pre-processing for nutation")
+def proc_nutation(s,fl=None):
+    logger.info(strm("pre-processing for p90 varied nutation"))
     orig_t = s.getaxis('t')
     s.set_units('p_90','s')
     s.reorder('t',first=True)
     s.chunk('t',['ph2','ph1','t2'],[2,4,-1])
     s.setaxis('ph2',r_[0.,2.]/4)
     s.setaxis('ph1',r_[0.,1.,2.,3.]/4)
     s.reorder('t2',first=False)
+    s.ft(['ph2','ph1'])
+    if fl is not None:
+        fl.next('after phase cycle FT')
+        fl.image(s)
+    s.ft('t2',shift=True)
+    if fl is not None:
+        fl.next('freq domain')
+        fl.image(s)
+    return s
+
+def proc_nutation_amp(s,fl=None):
+    logger.info(strm("pre-processing for amplitude varied nutation"))
+    orig_t = s.getaxis('t')
+    s.chunk('t',['ph2','ph1','t2'],[2,4,-1])
+    s.reorder(['ph1','ph2'])
+    s.setaxis('ph2',r_[0.,2.]/4)
+    s.setaxis('ph1',r_[0.,4.]/4)
+    s.set_units('t2','s')
+    s.set_units('amp','unknown')
```

Comment:
I would say this is just unitless? (b/c it's ratio relative to the max setting)

Reply:
I think you are referring to the units of am? so I changed unknown to unitless?

Reply:
yes, units of amp -- but you can just set the units to `None`